app/assets/stylesheets/layouts/_layouts-analysis.scss

```diff
@@ -14,7 +18,12 @@
   top: 0;
   bottom: 0;
   width: rem(336px);
-  height: 100%;
+  height: 100vh;
```

Comment:
But we don't really need it. I forgot to delete it. Now the layout is flex and we no longer need it.